src/components/Footer/Footer.jsx

```diff
@@ -0,0 +1,13 @@
+import styles from './Footer.module.css';
+import clsx from 'clsx';
+
+function Footer() {
+  return (
+    <footer className={styles.container}>
+      <img src="/images/logo-footer.png" alt="blockchain logo" />
+      <p className={clsx('body-short-02', styles.copyright)}>{new Date().getFullYear()} @Blockchain Education</p>
```

Comment:
`body-short-02` in an old typography. you would notice we added a comment to separate old typography.

Reply:
But why isn't it removed from the redesign?
Are we keeping it there for any reason?

Reply:
because we are making changes gradually. I could break other pages.
@vyktoremario